cookbooks/ey-backup/recipes/mysql.rb

```diff
@@ -41,7 +41,13 @@
 	end
 end
 
-if node.dna['backup_window'] != 0 && ['db_master','solo'].include?(node.dna['instance_role'])
+is_backup_instance = if node['db_backup']['mysql']['backup_slave']
```

Comment:
It took me a minute to get my head around this. Would it be clearer to assign `is_backup_instance` explicitly in the two different cases, rather than assigning the result of the if statement?  (It would have been clearer to me :D )